{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "val": 11
        },
        "etr": "",
        "start": "2020-12-23T13:49:03-0400"
      },
      "geom": {
        "p": [
          "gk{pGthtlK"
        ]
      },
      "id": "3",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
